playground/vite-react-ts/src/App.tsx

```diff
@@ -67,7 +67,7 @@ function ClerkProviderWithRoutes() {
   return (
     <ClerkProvider
       publishableKey={clerkPubKey}
-      navigate={to => navigate(to)}
+      push={to => navigate(to)}
```

Comment:
❓ Shall we pass replace ?
```suggestion
      push={to => navigate(to)}
      repalce={to => navigate(to, {replace: true} )}
```